<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #B9B4C7;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #6C667D;
	border-radius: 25px;
	color: #D0CBDC;
}

QPushButton:hover{
	background: #77679F;
	border-radius: 25px;
	color: #352F44;
}

QPushButton:pressed{
	background: #211D2B;
	border-radius: 25px;
	color: #B9B4C7;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="B_ToReg">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Зарегистрироваться!</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>416</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>480</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #352F44;
border-radius: 20px;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="10" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="B_Enter">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background: #6C667D;
	border-radius: 20px;
	color: #D0CBDC;
}

QPushButton:hover{
	background: #77679F;
	border-radius: 20px;
	color: #352F44;
}

QPushButton:pressed{
	background: #211D2B;
	border-radius: 20px;
	color: #B9B4C7;
}</string>
         </property>
         <property name="text">
          <string>Войти</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="T_Login">
         <property name="minimumSize">
          <size>
           <width>12</width>
           <height>39</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>437</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:  #D0CBDC;
color: #352F44;
border-radius: 20px; </string>
         </property>
         <property name="text">
          <string>Логин</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="LE_Login">
         <property name="minimumSize">
          <size>
           <width>12</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>437</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #B9B4C7;
color: #5F5675;
border-radius: 15px; </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="LE_Password">
         <property name="minimumSize">
          <size>
           <width>12</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>437</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #B9B4C7;
color: #5F5675;
border-radius: 15px; </string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="T_Password">
         <property name="minimumSize">
          <size>
           <width>12</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>437</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:  #D0CBDC;
color: #352F44;
border-radius: 20px; </string>
         </property>
         <property name="text">
          <string>Пароль</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="T_Enter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>438</width>
           <height>79</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>33</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #5C5470;
color: #B9B4C7;
border-radius: 30px; </string>
         </property>
         <property name="text">
          <string>Вход</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="L_Error">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:  #D0CBDC;
color: #6E677F;
border-radius: 20px; </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="L_YouHaventAcc">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>750</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #352F44;
border-radius: 20px;
color: #B9B4C7</string>
      </property>
      <property name="text">
       <string>Ещё нету аккаунта?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="B_Communication">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Отправить что-то на сервер</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="B_Close">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>52</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Свернуть</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="B_Exit">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>52</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
